Gaijinizer patch/Map408.txt

```diff
@@ -4,551 +4,687 @@
-
+> CONTEXT: Map408/events/1/pages/0/116/Dialogue/enemy29
+\\C[2]You must have realized that I'd make an
+appearance sooner or later with someone like
+\\N[1]-kun around who knows about "interventions"
+and has overcome "Death's Fate"...#Ron
 > END STRING
 
 > BEGIN STRING
 何度構えても無駄だよ。
 
 以前に比べて、ちょっとはマシになってはいるけど
 最初から、何度も何度も言ってるじゃないか。
-> CONTEXT: Map408/events/1/pages/0/123/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map408/events/1/pages/0/123/Dialogue/enemy29
+You can ready yourself all you like, but it's
+pointless. Though you've improved compared to
+before, I've had to tell you the same thing
+countless times.#RonIffy
 > END STRING
 
 > BEGIN STRING
 それでも、キミはボクに向かってくるつもりかい？
 
 そんなに、ボクが憎いのかい？
-> CONTEXT: Map408/events/1/pages/0/133/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map408/events/1/pages/0/133/Dialogue/enemy29
+Do you still intend on facing me knowing that?
 
+Do you hate me that much?#Ron
 > END STRING
 
 > BEGIN STRING
 当たり前だ……！！
 
 おまえこそ……
 いつまでそんなネコを被っているつもりだい……！
-> CONTEXT: Map408/events/1/pages/0/137/Dialogue/hyakki2 < UNTRANSLATED
+> CONTEXT: Map408/events/1/pages/0/137/Dialogue/hyakki2
+Of course I do...!!
 
+And what about you?! Just how long do you intend
+on wearing that cat...?!#Ron
 > END STRING
 
 > BEGIN STRING
 さっさと正体を現しな……！！
 
 \\C[10]『マッドハッター』\\C[0]………！！！！
-> CONTEXT: Map408/events/1/pages/0/143/Dialogue/hyakki2 < UNTRANSLATED
+> CONTEXT: Map408/events/1/pages/0/143/Dialogue/hyakki2
+Hurry up and show your true form...!!
 
+\\C[10]"Mad Hatter"\\C[0]...!!!#Ron
 > END STRING
 
 > BEGIN STRING
 ………………ボク、そっちの名前はキライだな
-> CONTEXT: Map408/events/1/pages/0/149/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map408/events/1/pages/0/149/Dialogue/enemy29
+......I'm not fond of that name.#Ron
 > END STRING
 
 > BEGIN STRING
 百鬼！！！！！！
-> CONTEXT: Map408/events/1/pages/0/186/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map408/events/1/pages/0/186/Dialogue/kazuya
+Hyakki!!!#Ron
 > END STRING
 
 > BEGIN STRING
 ………………大丈夫だよ。
 彼女は、この程度じゃ傷ひとつ付かない。
 
 キミに説明する間、邪魔しないでもらうためさ。
-> CONTEXT: Map408/events/1/pages/0/192/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map408/events/1/pages/0/192/Dialogue/enemy27
+...Don't worry, that wasn't enough to do any real
+damage. It was just so I could have the chance to
+explain things to you without interference.#Ron
 > END STRING
 
 > BEGIN STRING
 それに、『この世界』の『このタイミング』なら
 
 この方角に、妖精たちは一人もいない……
-> CONTEXT: Map408/events/1/pages/0/197/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map408/events/1/pages/0/197/Dialogue/enemy27
+And at this time in this world, there won't be
+any fairies here, either...#Ron
 > END STRING
 
 > BEGIN STRING
 僕の前に立っているのは、間違いなくチェシャ…
 
 だが、さっきまでのネコの悪魔の風貌ではなく
 どことなく不気味な人物のように見える……
-> CONTEXT: Map408/events/1/pages/0/203/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map408/events/1/pages/0/203/Dialogue
+The person standing in front of me is Cheshire without a
+doubt. But she no longer has the appearance of a cat
+demon, and for some reason looks quite eerie.#RonIffyWording
 > END STRING
```

Comment:
Minor rewording for flow:
```
The person standing in front of me was still Cheshire,
without a doubt. But instead of the feline demon I was
used to, she had a eerie appearance...
```